cmd/platform.go

```diff
@@ -251,3 +255,42 @@ func createGiteaClient(flag *flag.FlagSet, verifyFlags bool) (multigitter.Versio
 
 	return vc, nil
 }
+
+func createBitbucketServerClient(flag *flag.FlagSet, verifyFlags bool) (multigitter.VersionController, error) {
+	bitbucketBaseURL, _ := flag.GetString("base-url")
+	projects, _ := flag.GetStringSlice("org")
+	users, _ := flag.GetStringSlice("user")
+	repos, _ := flag.GetStringSlice("repo")
+
+	if verifyFlags && len(projects) == 0 && len(users) == 0 && len(repos) == 0 {
+		return nil, errors.New("no organization, user or repository set")
+	}
+
+	if bitbucketBaseURL == "" {
+		return nil, errors.New("no base-url set")
```

Comment:
Not sure what else to put other than maybe `no base-url set for bitbucket server`. I think when they select the `platform` of `bitbucket_server` it *should* be clear they we will need to supply a hosted bitbucket url. I will amend the message.

Reply:
Hopefully, it is. I have no experience with bitbucket, either cloud or on-prem. Before reading this PR I had no idea that it was different implementations/APIs. But hopefully it clear 🤞 . Otherwise it might be changed later if there is a lot of issues asking why the "bitbucket implementation is not working" 😉

Reply:
Agreed.